tests/test_unit.py

```diff
@@ -46,8 +58,56 @@ def test_interpreter_version(current_interpreter: Interpreter) -> None:
 
 
 def test_interpreter_version_info(current_interpreter: Interpreter) -> None:
-    assert current_interpreter.version_info() == (
-        sys.version_info[0],
-        sys.version_info[1],
-        sys.version_info[2],
-    )
+    assert current_interpreter.version_info() == sys.version_info[:3]
+
+
+def test_get_python_sitepackages() -> None:
+    assert get_python_sitepackages("python") == site.getsitepackages()
+
+
+@pytest.mark.skip(reason="We are unable to execute venv commands in tests")
```

Comment:
Why is this the case and is there value in keeping this test case if it's not runnable?

Reply:
This PR might help https://github.com/DataDog/riot/pull/128 but it needs #132 😢

Reply:
I wanted to basically leave it as a note as to why the test case is missing.

I suppose this isn't 100% the case, we'd have to setup the venv in order to test. Which is probably fine as you note in another comment about integration tests. I wasn't thinking of adding that at this time, but no reason not to.